### PR TITLE
docs(PPDSC-1832): Checkbox page

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-1832

**What**

1. Checkbox page

**I have done:**
 - [ ] Written unit tests against changes
 - [ ] Written accessibility tests against the component
 - [ ] Written functional tests against the component and/or NewsKit site
 - [ ] Updated relevant documentation
 - [ ] Updated the release notes for the [next release version](https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/2349695021/Release%2BNotes%2BCode%2B-%2BWeb%2BDocumentation)
 - [ ] Discussed with QA and/or design about any visual changes

**I have tested manually:**
 - [ ] Renders correctly on IE11
 - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
 - [ ] There are no accessibility errors reported by WAVE
 - [ ] The screen reader reads the expected text for the tabbable elements.
 - [ ] There are no new errors in the browser console coming from this PR.
 - [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
 - [ ] The Playground feature is working as expected

**Before**

**After:**

**Who should review this PR**

**How to test**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1832-checkbox-page 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-1832-checkbox-page/storybook

File: site/components/illustrations/components/form-illustration.tsx [hunk 1]
```diff
@@ -1,32 +1,114 @@
...
 import {Rect} from '../rect';
-import {Circle} from '../circle';
 import {Svg} from '../svg';
 
 export const FormIllustration: React.FC = () => (
   <Svg
-    width="1344"
-    height="759"
-    viewBox="0 0 1344 759"
+    width="1490"
```
Comment (mutebg) [inline site/components/illustrations/components/form-illustration.tsx:8]: I wonder if this file shod be inside 'form' folder?

File: site/pages/components/checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,1027 @@
...
+      components: [
+        {
+          title: 'FormInput Checkbox',
+          summary: (
+            <>
+              The FormInput Checkbox has a range of props that can be used to
+              define an appropriate experience for different use cases.Use this
+              component within the{' '}
+              <Link href="/components/form/">NewsKit Form component.</Link>
+            </>
```
Comment (mutebg) [inline site/pages/components/checkbox.tsx:698]: Can you please check 'radio-button.tsx' page, and add all these props to something like commonPropsRows and commonOverridesRows, I am pretty sure that Checkbox and FormInputCheckbox have almost identical props and overrides, so no need to have them twice.

File: site/pages/components/checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,1027 @@
...
+      },
+      introduction:
+        'Checkboxes are selection controls that allow users to select one or multiple items from a group of options. They typically appear in forms.',
+    }}
+    componentDefaultsKey="checkbox"
+    meta={{
+      status: MetaStatus.Supported,
+      introduced: 'v4.2.0',
+      introducedLink:
+        'https://github.com/newscorp-ghfb/ncu-newskit/releases/tag/v4.2.0',
```
Comment (mutebg) [inline site/pages/components/checkbox.tsx:42]: I wonder if we can just take the `v numver` and create the link something like
```
const introductionLink = `https://github.com/newscorp-ghfb/ncu-newskit/releases/tag/${introduced}`;

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @mutebg for reviewing the code :+1:

File: site/components/illustrations/components/form-illustration.tsx (hunk 1 above)
Comment (Srinivasan-Ramasamy) [reply site/components/illustrations/components/form-illustration.tsx]: Thanks @mutebg , I've moved into form folder.

File: site/pages/components/checkbox.tsx (hunk 2 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/checkbox.tsx:793]: Thanks @mutebg, I've updated overrides details used in const and I can see the confluence doc props have a difference  on Checkbox and FormInputCheckbox. Please suggest, how to proceed.

File: site/pages/components/checkbox.tsx (hunk 3 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/components/checkbox.tsx]: Thanks @mutebg, I've updated the changes.

**Review by mutebg — APPROVED** [review]

**Review by mutebg — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**